### PR TITLE
CAT-666 Link to Pathfinder service to get extremism alerts

### PR DESCRIPTION
**Pull request by steverendell**
Use pathfinder api instead of file

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -66,8 +66,9 @@ server:
     context-path: /
   forward-headers-strategy: native
   tomcat:
-    remote-ip-header: x-forwarded-for
-    protocol_header: x-forwarded-proto
```
Comment (steverendell) [inline src/main/resources/application.yml:70]: deprecated

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -87,6 +88,8 @@ management:
         time-to-live: 2000ms
       show-components: always
       show-details: always
+      probes:
+        enabled: true
```
Comment (steverendell) [inline src/main/resources/application.yml:92]: turn on liveness / readiness

File: src/main/java/uk/gov/justice/digital/hmpps/riskprofiler/services/ExtremismDecisionTreeService.java [hunk 3]
```diff
@@ -33,22 +31,19 @@ private ExtremismProfile decisionProcess(final String nomsId, final boolean prev
...
-                log.debug("extremism: {} Increased Risk of Extremism", nomsId);
-
                 if (previousOffences) {
-                    log.debug("extremism: {} has previous offences", nomsId);
+                    log.info("extremism: {} has previous offences", nomsId);
                     extremism.provisionalCategorisation("B");
                 } else {
                     extremism.provisionalCategorisation("C");
                 }
             } else {
```
Comment (yasinmustafa) [inline src/main/java/uk/gov/justice/digital/hmpps/riskprofiler/services/ExtremismDecisionTreeService.java:45]: minor but this `} else {` can be switched over to an `else  if` therefore avoiding the nested `if` block

**Review by yasinmustafa — APPROVED** [review]
lgtm

File: src/main/java/uk/gov/justice/digital/hmpps/riskprofiler/services/ExtremismDecisionTreeService.java (hunk 3 above)
Comment (steverendell) [reply src/main/java/uk/gov/justice/digital/hmpps/riskprofiler/services/ExtremismDecisionTreeService.java:45]: 👍

**Review by yasinmustafa — APPROVED** [review]
LGTM

**Pull request merged**